examples/2d_examples/FlowPastConstantHeatFluxCylinderCase/flow_past_constant_heat_flux_cylinder.py

```diff
@@ -0,0 +1,372 @@
+import elastica as ea
+import click
+import numpy as np
+import sopht.simulator as sps
+import sopht.utils as spu
+import sys
+
+sys.path.append("/")
```

Comment:
Why do we need this? This seems a bit dangerous. Same for the other files that have this